test/zwarpedarray.jl

```diff
@@ -1,15 +1,18 @@
 using ZWarpedArrays, Images, CoordinateTransformations
-using Base.Test
+using Test
 
 img0 = zeros(Normed{UInt16, 16}, 5,5,3,3)
 imgpost = zeros(5,5,3,3)
-normed1 = Normed{UInt16,16}(1.0)
-imgpost[3,3,:,:] = normed1 #We hope the warped array looks like this
-img0[3,3,1,:] = normed1 #centered
-img0[4,3,2,:] = normed1 #shifted +1 in x
-img0[4,4,3,:] = normed1 #shifted +1 in x and y
+normed1 = reinterpret(Normed{UInt16,16}, UInt16(2^8))
```

Comment:
Can always use `N0f16` as a shorthand for `Normed{UInt16,16}`. Out of curiosity, why did you switch the value? It's now a misleading name, since 
```julia
julia> normed1 = reinterpret(Normed{UInt16,16}, UInt16(2^8))
0.00391N0f16
```
(`normed1` = 1.0 expressed as a normed type)

`N0f16(1.0)` still works fine.